.github/workflows/pr.yaml

```
@@ -342,6 +352,10 @@ jobs:
         node_type: "cpu4"
         build_type: pull-request
         run_script: "ci/cudf_pandas_scripts/pandas-tests/diff.sh"
+  spark-rapids-jni:
+    needs: changed-files
+    uses: ./.github/workflows/spark-rapids-jni.yaml
+    if: fromJSON(needs.changed-files.outputs.changed_file_groups).test_spark
```

I think we will always want to test JNI and Spark-RAPIDS with the same changed-files criteria. I'm a bit confused why this conversation ended up with two copies of the same criteria. https://github.com/rapidsai/cudf/pull/17781#discussion_r1936141574
```suggestion
    if: fromJSON(needs.changed-files.outputs.changed_file_groups).test_java
```